<ui version="4.0" >
 <class>dialog_profilemodify</class>
 <widget class="QDialog" name="dialog_profilemodify" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>367</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>450</width>
    <height>367</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>450</width>
    <height>367</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Edit Profile</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/resources.qrc" >
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <property name="locale" >
   <locale country="UnitedKingdom" language="English" />
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Profile info</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_14" >
        <property name="minimumSize" >
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>Profile name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="profileName" >
        <property name="maxLength" >
         <number>64</number>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>label_14</zorder>
     <zorder>profileName</zorder>
     <zorder>groupBox</zorder>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Game server</string>
     </property>
     <layout class="QFormLayout" name="formLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="minimumSize" >
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>Server IP:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="CIPTextBoxWidget" native="1" name="serverIP" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="minimumSize" >
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>Game port:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="serverGamePort" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="buttonSymbols" >
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>65535</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Admin settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="minimumSize" >
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>HTTP port:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="serverHttpPort" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="buttonSymbols" >
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="minimumSize" >
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="serverPassword" >
        <property name="inputMask" >
         <string/>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="maxLength" >
         <number>32</number>
        </property>
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QCheckBox" name="displayPassword" >
        <property name="text" >
         <string>Display password</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CIPTextBoxWidget</class>
   <extends>QWidget</extends>
   <header>interface/IPTextBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections/>
</ui>
